certified-connectors/EBMS/apiDefinition.swagger.json

```diff
@@ -0,0 +1,481 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "EBMS",
+    "description": "Connect your EBMS data with hundreds of other services! Contact your Eagle Business Software Account Manager to activate your connector.",
+    "version": "1.0",
+    "contact": {
+      "name": "Eagle Business Software",
+      "url": "https://info.eaglebusinesssoftware.com",
+      "email": "info@eaglebusinesssoftware.com"
+    }
+  },
+  "host": "ecc.101010101010101.servicebus.windows.net",
```

Comment:
@Amjed-Ayoub this host value is not a valid URL. This is intentional, because each of our clients will have their own personalized URL. This value is simply a placeholder. It does not point to a dev or test environment.

Please advise if there's something we should do with it.

Reply:
@bradleymarshall1982 If I understand correctly, you are trying to set a dynamic host URL for your connector. Please see our docs on how to [Set Host URL | Microsoft Learn](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Flearn.microsoft.com%2Fen-us%2Fconnectors%2Fcustom-connectors%2Fpolicy-templates%2Fdynamichosturl%2Fdynamichosturl&data=05%7C01%7Cv-amjedayoub%40microsoft.com%7C6608c2edc91a441a973f08dacd8e6ec5%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638048309238464455%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=A3BeY89Q2pEzr0GmtOInIynk5qFC6xzOliFX6xC1GkQ%3D&reserved=0).
May I ask you please to take a look at [Set HTTP Header | Microsoft Learn](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Flearn.microsoft.com%2Fen-us%2Fconnectors%2Fcustom-connectors%2Fpolicy-templates%2Fsetheader%2Fsetheader&data=05%7C01%7Cv-amjedayoub%40microsoft.com%7C6608c2edc91a441a973f08dacd8e6ec5%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638048309238620695%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=EnbX%2BqF2fRXsu6cnJityCwOWGekWYY3zg8UkGURWRFY%3D&reserved=0) and [Set Query String Parameter | Microsoft Learn](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Flearn.microsoft.com%2Fen-us%2Fconnectors%2Fcustom-connectors%2Fpolicy-templates%2Fsetqueryparameter%2Fsetqueryparameter&data=05%7C01%7Cv-amjedayoub%40microsoft.com%7C6608c2edc91a441a973f08dacd8e6ec5%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638048309238620695%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=RuMRHxIeg4TOvhRUwwjqK9HZr5eZ%2B9mKScpL7SJ8udw%3D&reserved=0).
 
The docs above provide some examples from an existing connectors which use these features.
 
I hope this helps.
 
I look forward to hearing from you.
 
Thank you so much again.

Reply:
@Amjed-Ayoub We are already using connection parameters to specify dynamic host URL in apiProperties.json. I changed this **host** URL to clarify.